services/workers/src/test/java/org/sagebionetworks/schema/worker/SynpaseSchemaBootstrapWorkerTest.java

```diff
@@ -1,21 +1,48 @@
 package org.sagebionetworks.schema.worker;
 
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.sagebionetworks.repo.manager.schema.JsonSchemaManager;
+import org.sagebionetworks.repo.model.schema.JsonSchemaInfo;
+import org.sagebionetworks.repo.model.schema.ListJsonSchemaInfoRequest;
+import org.sagebionetworks.repo.model.schema.ListJsonSchemaInfoResponse;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
 @ExtendWith(SpringExtension.class)
 @ContextConfiguration(locations = { "classpath:test-context.xml" })
 public class SynpaseSchemaBootstrapWorkerTest {
+	
+	public static final long MAX_WAIT_MS = 60*1000*2;
 
 	@Autowired
 	private JsonSchemaManager jsonSchemaManager;
 	
 	@Test
-	public void testRun() {
-		
+	public void testRun() throws InterruptedException {
```

Comment:
We might want to truncate the data in a before to increase the change that it the worker to put it there?